<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox_Type">
     <item>
      <property name="text">
       <string>Annuity</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Differentiated</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_InterestRate">
     <property name="text">
      <string>Interest Rate</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_total_credit_amount">
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>1000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_interest_rate">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton_Calculate">
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_TotalPayment">
     <property name="text">
      <string>Total Payment</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_TotalPayment">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>1000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_MonthlyPayment">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>1000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Term">
     <property name="text">
      <string>Term</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_Overpayment">
     <property name="text">
      <string>Overtayment on credit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_prc">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_MonthlyPayment">
     <property name="text">
      <string>Monthly payment</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_TotalCreditAmount">
     <property name="text">
      <string>Total Credit Amount</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_Term">
     <item>
      <property name="text">
       <string>Months</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Years</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_Overpayment">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>1000000000000000000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox_term">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>600</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>doubleSpinBox_total_credit_amount</tabstop>
  <tabstop>spinBox_term</tabstop>
  <tabstop>comboBox_Term</tabstop>
  <tabstop>doubleSpinBox_interest_rate</tabstop>
  <tabstop>comboBox_Type</tabstop>
  <tabstop>pushButton_Calculate</tabstop>
  <tabstop>doubleSpinBox_TotalPayment</tabstop>
  <tabstop>doubleSpinBox_Overpayment</tabstop>
  <tabstop>doubleSpinBox_MonthlyPayment</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
